libs/typescript/balance-verification-utils/utils.ts

```diff
@@ -0,0 +1,29 @@
+type Network = 'mainnet' | 'goerli' | 'sepolia';
+type Address = string;
+type Timestamp = string;
+type AddressConfig = { [key in Network]: Address };
+type TimestampConfig = { [key in Network]: Timestamp };
+
+const lidoWithdrawalCredentials: AddressConfig = {
+  mainnet: '0x010000000000000000000000b9d7934878b5fb9610b3fe8a5e441e8fad7e293f',
+  goerli: '0x010000000000000000000000dc62f9e8C34be08501Cdef4EBDE0a280f576D762',
+  sepolia: '0x010000000000000000000000De7318Afa67eaD6d6bbC8224dfCe5ed6e4b86d76',
+};
```

Comment:
I suggest placing this information in a file named `constants.ts`. However, considering whether these should be classified as constants, it's worth pondering if there's a chance that withdrawal credentials might be assigned to different addresses in the future. If that's a possibility, perhaps it would be more flexible to read these values from the environment instead.

Also some kind of documentation about the origin of the addresses might be a good idea